.travis.yml

```diff
@@ -25,7 +25,7 @@ before_install:
 
 install:
     - |
-      MINICONDA_URL="http://repo.continuum.io/miniconda"
+      MINICONDA_URL="https://repo.continuum.io/miniconda"
```

Comment:
Thanks for pointing this out @ocefpaf :smiley: 

@bjlittle I have used conda-smithy and have a PR up (see PR #2 ) that you will need to rebase onto